tests/token/erc721/test_ERC721_Mintable_Burnable.py

```diff
@@ -1,48 +1,34 @@
 import pytest
 import asyncio
 from starkware.starknet.testing.starknet import Starknet
-from utils import Signer, uint, str_to_felt, MAX_UINT256, assert_revert
 from starkware.starkware_utils.error_handling import StarkException
 from starkware.starknet.definitions.error_codes import StarknetErrorCode
-
+from utils import (
+    Signer, str_to_felt, ZERO_ADDRESS, TRUE, FALSE, assert_revert,
+    get_contract_def, cached_contract, to_uint, sub_uint, add_uint
+)
 
 signer = Signer(123456789987654321)
 
-ZERO_ADDRESS = 0
-
-# bools (for readability)
-false = 0
-true = 1
-not_bool = 2
-
-# random user addresses
-user1 = 123
-user2 = 234
-user3 = 345
-user4 = 456
-user5 = 567
-
-# random uint256 tokenIDs
-first_token_id = (5042, 0)
-second_token_id = (7921, 1)
-third_token_id = (0, 13)
-fourth_token_id = MAX_UINT256
-fifth_token_id = (234, 345)
-sixth_token_id = (9999, 9999)
-seventh_token_id = (987, 654)
-eighth_token_id = (445, 101)
-other_owned_token = (123, 321)
-nonexistent_token = (111, 222)
-token_to_burn = (12345, 6789)
-
+NONEXISTENT_TOKEN = to_uint(999)
+# random token IDs
+TOKENS = [to_uint(5042), to_uint(793)]
+# test token
+TOKEN = TOKENS[0]
+# random user address
+RECIPIENT = 555
 # random data (mimicking bytes in Solidity)
-data = [0x42, 0x89, 0x55]
-
+DATA = [0x42, 0x89, 0x55]
 # random URIs
-sample_uri = [
-    str_to_felt('mock://mytoken.v1'),
-    str_to_felt('mock://mytoken.v2')
-]
+SAMPLE_URI_1 = str_to_felt('mock://mytoken.v1')
+SAMPLE_URI_2 = str_to_felt('mock://mytoken.v2')
+
+# selector ids
+IERC165_ID = 0x01ffc9a7
+IERC721_ID = 0x80ac58cd
+IERC721_METADATA_ID = 0x5b5e139f
```

Comment:
are these used anywhere else? it sounds like they could be centralized somewhere like `constants.cairo` (if used from contracts) or another common module.

Reply:
Hmm that's a good question, the `IERC165_ID` is definitely used elsewhere. Yeah, I'll write an issue to move interface ids in Cairo to `constants`. On a more macro note, what do you think about having a `utilities` directory with:
1. constants, math methods, and assert methods in `utils.py` 
2. all `Signer` methods in something like `Signer.py`?